src/commands/appSettings/uploadAppSettings.ts

```diff
@@ -53,6 +53,12 @@ export async function uploadAppSettingsInternal(context: IActionContext, client:
         }
 
         const excludedAppSettings: string[] = [];
+        // Don't want AzureWebJobsStorage to be overwritten in this case as it should point at a live resource: https://github.com/microsoft/vscode-azurefunctions/issues/3298
+        if (localSettings.Values["AzureWebJobsStorage"] === 'UseDevelopmentStorage=true') {
```

Comment:
Small nit, but I think there's a constant we've been using for `UseDevelopmentStorage=true`